### PR TITLE
Code-owners

### PR DESCRIPTION
**Pull request by terrikwak**
## Proposed changes

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

**Review by aidan-kwon — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
 #*       @global-owner1 @global-owner2
-*       @kjhman21 @terrikwak @dcground @neoofklaytn @aidan-kwon @terry-klaytn
+*       @kjhman21 @terrikwak @neoofklaytn @aidan-kwon @terry-klaytn @toniya-klaytn @tantely-klaytn @praveen-klaytn
```
Comment (terrikwak) [inline .github/CODEOWNERS:9]:
```suggestion
*       @kjhman21 @terrikwak @neoofklaytn @aidan-kwon @terry-klaytn @toniya-klaytn @tantely-klaytn @praveen-klaytn @AyDeveloper 
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -6,7 +6,7 @@
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
 #*       @global-owner1 @global-owner2
-*       @kjhman21 @terrikwak @dcground @neoofklaytn @aidan-kwon @terry-klaytn
+*       @kjhman21 @terrikwak @neoofklaytn @aidan-kwon @terry-klaytn @toniya-klaytn @tantely-klaytn @praveen-klaytn @AyDeveloper
```
Comment (kjhman21) [inline .github/CODEOWNERS:9]: @terrikwak Seems like @AyDeveloper does not have access or wrong ID.

Comment (terrikwak) [inline .github/CODEOWNERS:9]:
```suggestion
*       @kjhman21 @terrikwak @neoofklaytn @aidan-kwon @terry-klaytn @toniya-klaytn @tantely-klaytn @praveen-klaytn @ayo-klaytn
```

Comment (terrikwak) [reply .github/CODEOWNERS]: @kjhman21 Yes, I requested Tony to help

**Review by kjhman21 — APPROVED** [review]

**Pull request merged**